src/constEval.ts

```diff
@@ -681,9 +741,9 @@ export function evalConstantExpression(
         case "string":
             return ensureString(interpretEscapeSequences(ast.value), ast.loc);
         case "op_unary":
-            return evalUnaryOp(ast.op, ast.operand, ast.loc, ctx);
+            return fullyEvalUnaryOp(ast.op, ast.operand, ast.loc, ctx);
```

Comment:
maybe we should rename everything to `fullyEvalX` in the `evalConstantExpression` function? for consistency reasons, like `evalMethod` -> `fullyEvalMethod`, etc. We can put a comment that it's ongoing work to make all of those actually fully evaluate